src/zenml/integrations/skypilot_aws/__init__.py

```diff
@@ -31,7 +31,7 @@ class SkypilotAWSIntegration(Integration):
     """Definition of Skypilot AWS Integration for ZenML."""
 
     NAME = SKYPILOT_AWS
-    REQUIREMENTS = ["skypilot[aws]<=0.4.1"]
+    REQUIREMENTS = ["skypilot[aws]"]
```

Comment:
Just out of curiosity, why did we remove the version limitations from all the skypilot orchestrators?

Also, would it make sense to at least use "skypilot[xxx]<=whatever the max version is"? This way, if there is a breaking skypilot release between ZenML releases the mismatch can not cause any issues.